src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -459,6 +459,23 @@ protected override void OnDataContextEndUpdate()
             }
         }
 
+        /// <summary>
+        /// Handles 'select all' (Ctrl+A) in <see cref="SelectingItemsControl"/>.
+        /// </summary>
+        /// <param name="e">The key events.</param>
+        protected override void OnKeyDown(KeyEventArgs e)
+        {
+            base.OnKeyDown(e);
+
+            if (!e.Handled)
+            {
+                if (this.SelectionMode == SelectionMode.Multiple && e.Key == Key.A && e.Modifiers == InputModifiers.Control)
```

Comment:
I think this is not the proper way to handle hotkeys - look at https://github.com/AvaloniaUI/Avalonia/blob/211aaaa6e1033d79b24ebca3f170636955d5b836/src/Avalonia.Controls/TextBox.cs#L375-L384 for hotkey implementation.

Reply:
Miss this, thanks.
PS: so, ListBox use Control and Shift hotkeys, can i fix this too?